jenkins/Dockerfile

```diff
@@ -6,3 +6,7 @@ ENV JAVA_OPTS "-Dhudson.model.DirectoryBrowserSupport.CSP="
 
 COPY plugins.txt /usr/share/jenkins/ref/plugins.txt
 RUN /usr/local/bin/plugins.sh /usr/share/jenkins/ref/plugins.txt
+
+USER root
```

Comment:
Was this `USER root` needed? If yes, then doesn't it need `USER`'ing back to a non-root user?

Reply:
I have moved it from the original file. I have not tested without.

Reply:
I have done further tests.
* Removing ``USER root`` leads to ``/var/jenkins_home/copy_reference_file.log: Permission denied``
* Switching to another user e.g. ``omero`` to ``devspace_jenkins_1  Cannot start service jenkins: linux spec user: unable to find user omero``